google-cloud-core/pom.xml

```diff
@@ -40,6 +40,22 @@
       <scope>compile</scope>
     </dependency>
     <dependency>
+      <groupId>com.google.http-client</groupId>
+      <artifactId>google-http-client</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>com.google.code.findbugs</groupId>
+          <artifactId>jsr305</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
```

Comment:
I didn't look into the details.  But google-http-client defines findbugs as a compile dependency.  A compile dependency of a compile dependency is a transitive compile dependency. So I think it's safe to make it compile time dependency.